### PR TITLE
Create Collapse Kit - Refresh

### PR DESCRIPTION
**Pull request by coleerikson**
#### Screens
collapse-open-white

collapse-closed-white

collapse-in-nitro

collapse-open-dark

collapse-closed-dark

#### Breaking Changes

No, new kit.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/teams/258/backlog/sprints/closed?sprint_id=6438#open-backlog-item-40493

#### How to test this

- Check New Kit Docs
- Specs written

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.

### CODE REVIEW

File: playbook/app/pb_kits/playbook/_playbook.scss [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+@import 'pb_background/background';
+@import 'pb_body/body';
+@import 'pb_button/button';
 @import 'pb_button_toolbar/button_toolbar';
-@import "pb_caption/caption";
-@import "pb_card/card";
-@import "pb_title/title";
+@import 'pb_caption/caption';
+@import 'pb_card/card';
+@import 'pb_title/title';
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/_playbook.scss:8]: It looks like these are generated with single quotes normally. I wonder how did the left side get doubles?

File: playbook/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   "author": "powerhome",
   "license": "ISC",
   "engines": {
-    "node": "10.17.0 || 12.20.0",
+    "node": "10.17.0 || 12.20.1",
```
Comment (thestephenmarshall) [inline playbook/package.json:24]: I thought this was one that had been updated upstream earlier by Jasper for GH Actions, but that may not have made it in afterall.

**Review by thestephenmarshall — COMMENTED** [review]
Looking pretty good. Let's see about adding some tests for the JSX parts too.

**Comment by kre8sions** [comment]
This looks good

**Pull request merged**